src/schema/v2/partner/partnerInquiryRequest.ts

```diff
@@ -40,7 +40,7 @@ export const InquiryRequestType = new GraphQLObjectType<any, ResolverContext>({
               : `• ${question?.question}`
           )
         })
-        if (message) lines.unshift([message, "\n"].join())
```

Comment:
My only theory is that `join` behaves differently in JS than it does in Ruby. In Ruby this would have just added a new line to the `message` string but in JS it inserts that comma. Maybe this was just a mistake? 🤷

Reply:
Agreed!

And for a bit of context: the `formattedFirstMessage` didn't even exist in convos before the revamp (it was added as a workaround to include inquiry questions, which used to come directly from the email message in convos v1).

I understand its only purpose today is to be used to present those messages containing the inquiry question.
And looking at the examples on CMS convos on staging today for conversations containing inquiry questions, the comma really seems weird (much better without it)

Reply:
That's right. The `join` method adds a comma between elements in the array, unless you specify another string to use (as we do with `join("\n")` on the next line).

I think that this was just an oversight.